.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 node_modules
 cjs
 es
+*.swp
```

Comment:
What's this for?

Reply:
Ignoring vim swap files / editor created files